packages/react-app/src/App.jsx

```diff
@@ -325,6 +345,9 @@ function App() {
             <Route path="/challenge/:challengeId">
               <ChallengeDetailView userObject={userObject} serverUrl={serverUrl} address={address} />
             </Route>
+            <Route path="/challenge-review" exact>
```

Comment:
Forgot to mention this!

Even though we won't show the link to non-admin users, we should probably prevent other users from just hitting `/challenge-review` and accessing the page.

There's [a guide on react router docs](https://reactrouter.com/web/example/auth-workflow) to implement private routes. Have a look at lines 50-52 (component usage) and 135-156 (component declaration)

Reply:
Even if they access the page, the backend endpoint should return a 401.

Reply:
Yes, we can leave a todo as in the link above

Reply:
I added a todo for that, but I dont really understand how protecting a route like this on the frontend is useful 🤔 As long as it's not a route where users could be led by a mistake

Reply:
I think it's just an extra caution. The cost is very low and the benefits are also low, but in some situations this could prevent bad stuff from happening.

I think of this as another door people will have to cross to mess with admin stuff. We have the middleware (strong door) and will have the link hiding (weaker door). There's no harm in adding an extra door (also weak), given the cost. Doors could be open due to bugs, or people trying to force their way in.

If this meant a 2 days refactor I'd say it's not worth it because what we have in place should be enough.

I hope it makes sense 🙃